<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>212</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>561</width>
    <height>212</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>561</width>
    <height>212</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="Chart">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>257</width>
       <height>212</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>257</width>
       <height>212</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : rgba(255, 255, 255,1);
border-radius:5px;
</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="ChartPage1">
      <property name="minimumSize">
       <size>
        <width>257</width>
        <height>212</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>257</width>
        <height>212</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgba(255, 255, 255,1);
border-radius:5px;
</string>
      </property>
      <layout class="QVBoxLayout" name="ChartPage1Layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="ChartPage2">
      <property name="minimumSize">
       <size>
        <width>257</width>
        <height>212</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>257</width>
        <height>212</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgba(255, 255, 255,1);
border-radius:5px;
</string>
      </property>
      <layout class="QVBoxLayout" name="ChartPage2Layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="TabWidget">
     <property name="minimumSize">
      <size>
       <width>278</width>
       <height>212</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>278</width>
       <height>212</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(191,191,191,0);
color : black;
border-radius:5px;
</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Price">
      <property name="minimumSize">
       <size>
        <width>276</width>
        <height>190</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>276</width>
        <height>190</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgba(191, 191, 191,1);
border-radius:5px;</string>
      </property>
      <attribute name="title">
       <string>시세</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="cursorLayout">
         <item>
          <widget class="QWidget" name="coordinate" native="true">
           <property name="minimumSize">
            <size>
             <width>123</width>
             <height>49</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>123</width>
             <height>49</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color : rgba(255, 255, 255,1);
border-radius:5px;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="coordinateLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="price" native="true">
           <property name="minimumSize">
            <size>
             <width>122</width>
             <height>49</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>122</width>
             <height>49</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color : rgba(255, 255, 255,1);
border-radius:5px;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="priceLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="targetLayout">
         <item>
          <widget class="QWidget" name="target" native="true">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>49</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>49</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color : rgba(255, 255, 255,1);
border-radius:5px;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="target_label">
              <property name="text">
               <string>목표 시세</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="target_price" native="true">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>49</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>49</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color : rgba(255, 255, 255,1);
border-radius:5px;</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="target_price_text">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="setButton">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>49</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>49</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(0, 21, 80, 170);
color : white;
border-radius:5px;
</string>
           </property>
           <property name="text">
            <string>설정</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="togglebutton">
         <property name="minimumSize">
          <size>
           <width>254</width>
           <height>52</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>254</width>
           <height>52</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0, 21, 80, 170);
color : white;
border-radius:5px;
</string>
         </property>
         <property name="text">
          <string>월봉 차트 변환</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="News">
      <property name="minimumSize">
       <size>
        <width>276</width>
        <height>190</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>276</width>
        <height>190</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgba(191, 191, 191, 1);
border-radius:5px;</string>
      </property>
      <attribute name="title">
       <string>뉴스</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QScrollArea" name="NewsScrollArea">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="NewsScrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>258</width>
            <height>172</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="NewsScrollLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Research">
      <property name="minimumSize">
       <size>
        <width>276</width>
        <height>190</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>276</width>
        <height>190</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgba(191, 191, 191, 1);
border-radius:5px;</string>
      </property>
      <attribute name="title">
       <string>리서치</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QScrollArea" name="ResearchScrollArea">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="ResearchScrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>258</width>
            <height>172</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="ResearchScrollLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PublicNotice">
      <property name="minimumSize">
       <size>
        <width>276</width>
        <height>190</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>276</width>
        <height>190</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgba(191, 191, 191,1);
border-radius:5px;</string>
      </property>
      <attribute name="title">
       <string>공시</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QScrollArea" name="PublicNoticeScrollArea">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="PublicNoticeScrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>258</width>
            <height>172</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="PublicNoticeScrollLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
